# HG changeset patch
# Parent 4d39a980152bab88c02a472bf76156da3ce93151
# User Jason Kim <jasonwkim@google.com>
With getLazyBitcodeModule, uses and defs don't line up. Load the whole bitcode at once.

diff -r 4d39a980152b tools/lto/LTOModule.cpp
--- a/tools/lto/LTOModule.cpp	Tue Aug 09 16:42:05 2011 -0700
+++ b/tools/lto/LTOModule.cpp	Tue Aug 09 16:42:06 2011 -0700
@@ -144,8 +144,8 @@
   }
 
   // parse bitcode buffer
-  OwningPtr<Module> m(getLazyBitcodeModule(buffer, getGlobalContext(),
-                                           &errMsg));
+  OwningPtr<Module> m(ParseBitcodeFile(buffer, getGlobalContext(), // @LOCALMOD
+                                       &errMsg));
   if (!m) {
     delete buffer;
     return NULL;
